extensions/timeline/EasyTimeline.pl

```diff
@@ -4654,6 +4654,10 @@ sub ValidDateRange {
 sub DateMedium {
     my $from = shift;
     my $till = shift;
+    if (! defined $from || ! defined $till ) {
+    	&Error2("from ($from) or till ($till) is not defined, Returning default to prevent infinite loop");
+    	return  sprintf("%02d/%02d/%04d", 1, 1, 1800);
```

Comment:
That is a bit over engineered :P Could just return "01/01/1800"
```suggestion
    	return  "01/01/1800";
```